datajunction-server/datajunction_server/api/users.py

```diff
@@ -0,0 +1,91 @@
+"""
+User related APIs.
+"""
+
+from typing import List, Optional
+
+from fastapi import Depends, Query
+from sqlalchemy import distinct, func, select
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import joinedload, selectinload
+
+from datajunction_server.database.column import Column
+from datajunction_server.database.history import ActivityType, EntityType, History
+from datajunction_server.database.node import Node, NodeRevision
+from datajunction_server.database.user import User
+from datajunction_server.internal.access.authentication.http import SecureAPIRouter
+from datajunction_server.models.node import NodeMinimumDetail
+from datajunction_server.models.user import UserActivity, UserOutput
+from datajunction_server.utils import get_current_user, get_session, get_settings
+
+settings = get_settings()
+router = SecureAPIRouter(tags=["users"])
+
+
+@router.get("/users/me", response_model=UserOutput)
```

Comment:
Isn't this is a duplicate of the `/whoami` endpoint? If we add this here we should remove that one.

Reply:
Good point, I'll remove this one